lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AddressService.java

```diff
@@ -55,8 +55,7 @@ public List<CanonicalAddress> parseAddresses(String addressBody)
         try {
             addresses = getAddressReader().readValue(addressBody);
         } catch (JsonProcessingException e) {
-            throw new AddressProcessingException(
-                    "Could not parse addresses to a valid CanonicalAddress");
+            throw new AddressProcessingException("could not parse addresses..." + e.getMessage());
```

Comment:
Maybe make this error more meaningful e.g. `"Jackson could not parse addresses to a List of CanonicalAddress": + e.getMessage();`?